argoverse/evaluation/eval_tracking.py

```diff
@@ -104,176 +104,149 @@ def get_distance(x1: Dict[str, np.ndarray], x2: Dict[str, np.ndarray], name: str
         raise NotImplementedError("Not implemented..")
 
 
-def eval_tracks(
+def get_frame_data(
```

Comment:
hey @ahuller3, how long does `get_frame_data()` take compared to the mot metrics eval function call?